lib/commands/logging.js

```diff
@@ -38,9 +40,9 @@ extensions.supportedLogTypes = {
   server: {
     description: 'Appium server logs',
     getter: (self) => {
-      if (!self.relaxedSecurityEnabled) {
-        throw new Error('Appium server must have relaxed security flag set ' +
-                        'in order to retrieve server logs');
+      if (!self.isFeatureEnabled(SERVER_LOGS_FEAT_NAME)) {
+        throw new Error(`Security level does not allow feature '${SERVER_LOGS_FEAT_NAME}'. ` +
+                        `See Appium documentation for allowing insecure features.`);
```

Comment:
Can this link to the actual docs?

Reply:
Yes, will work on actual docs. Also will write a function that simply takes a feature name and description and generates this error message, so they all have the same format.

Reply:
That sounds great!

Reply:
`ensureFeatureEnabled` links to actual docs.